### PR TITLE
Display project specific values in lanes table

### PR DESCRIPTION
**Pull request by sofiahag**
Compare with the values in each flowcells page, project yields, these should be the same. 

I think the Q30 value is rounded off in a slightly different way, I was hoping Chuan would have better insight as to why this is :-D 

Maybe some code is redundant now and can be cleaned up a bit, I added the WIP label for this reason.

### CODE REVIEW

**Comment by sofiahag** [comment]
Ok, I don't know if this classifies as cleaning up the code, but it was the best I could do 😆

File: ngi_reports/utils/entities.py [hunk 1]
```diff
@@ -553,17 +562,17 @@ def populate(self, log, organism_names, **kwargs):
...
-                self.samples_unit = 'Mreads'
-                samples_divisor = 1000000
-            else:
-                self.samples_unit = 'Kreads'
-                samples_divisor = 1000
-        for sample in self.samples:
-            self.samples[sample].total_reads = '{:.2f}'.format(self.samples[sample].total_reads/float(samples_divisor))
-
+        samples_divisor = 1000000 if max_total_reads > 1000000 else 1000 if max_total_reads > 1000 else 1
+        for sample_obj in self.samples.values():
```
Comment (aanil) [inline ngi_reports/utils/entities.py:566]: What happened to `samples_unit`?

File: ngi_reports/utils/entities.py [hunk 2]
```diff
@@ -96,6 +96,7 @@ def __init__(self):
         self.aborted_samples = OrderedDict()
         self.samples = OrderedDict()
         self.flowcells = {}
+        self.proj_lane_info = {}
```
Comment (aanil) [inline ngi_reports/utils/entities.py:99]: Why can't this be included in `self.flowcells`? We are looping through it anyway in `project_summary.md`

**Review by aanil — APPROVED** [review]
😄

**Pull request merged**